src/csharp/Extensions/Microsoft.Spark.Extensions.Hyperspace.E2ETest/Resources/sample-data.csv

```diff
@@ -0,0 +1,11 @@
+﻿c1,c2,c3,c4,c5
```

Comment:
Can we use the existing resources for E2E or no? If not, you can also create a simple df using `CreateDataFrame` (just one row), write a file and use it.

Reply:
Removed.  Will reuse from E2E tests.